incubator/taiga/values.yaml

```diff
@@ -0,0 +1,132 @@
+### Default values for taiga.
+
+image:
+  repository: mvitale1989/docker-taiga
```

Comment:
This image appears to be a new on (for this?) that's manually created. What's the plan for updates and maintenance?

Reply:
That's a fork of [benhutchins/docker-taiga](https://github.com/benhutchins/docker-taiga), which i slightly adapted for clean usage in Kubernetes (added option of using as an init container, and implemented proper termination signal handling). My goal is merging the changes back in the original repository, for which i already have a PR open, and switch back to that image after the merge; but depending on how this process goes i might maintain the fork with these adaptations, and merge upstream changes back into it.

Reply:
It appears https://github.com/benhutchins/docker-taiga doesn't have an up to date public image on docker hub. Once your changes land where will they be shared with others? How will things be kept up to date?

Reply:
The author has set an automated build job for the repo on [docker hub](https://hub.docker.com/r/benhutchins/taiga/builds/). The builds failed for a while due to an error that my PR resolves, as you can see in [my automated builds](https://hub.docker.com/r/mvitale1989/docker-taiga/builds/), so once the changes are merged upstream, the latest image will be publicly available.

Reply:
The author of the original image has been unresponsive so far. Given my repo also diverged substantially in the provisioning approach, i will keep using and maintaining my own image.